src/test/java/io/gingersnapproject/testcontainers/TestContainerHelper.java

```diff
@@ -0,0 +1,54 @@
+package io.gingersnapproject.testcontainers;
+
+import static io.gingersnapproject.testcontainers.BaseGingersnapResourceLifecycleManager.TESTING_RESOURCES;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import io.gingersnapproject.testcontainers.hotrod.HotRodContainer;
+
+import io.restassured.response.ValidatableResponse;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+public class TestContainerHelper {
+
+   public static void executeSqlStatements(String rule, String ... sql) {
```

Comment:
If we don't use the `TESTING_RESOURCES` approach, I think we should just add this method to a `JdbcUtil` class and pass the `Connection` as a parameter.

Reply:
I created a base class for when writing tests that need to interact with the containers using the test injector from your suggestion.